crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -883,6 +884,40 @@ impl Object {
         }
     }
 
+    /// Fetch objects by their id and version. If you need to query for live objects, use the
```

Comment:
nit: ordering, I think we put the query functions above the paginate functions usually.